data/virt_autotest/virt_logs_collector.sh

```diff
@@ -354,6 +354,7 @@ function collect_extra_logs_from_host() {
 	local libvirt_daemon_logs="${libvirt_log}/*d.log"
 	local xen_log="/var/log/xen"
 	local xen_boot_log="${xen_log}/xen-boot.log"
+        local kernel_log="/var/log/kern.log"
```

Comment:
It is less than 1M when failure happens at the last step. kern.log is in extra logs folder, it is not collected by supportconfig by default. Since its size is not big, so it will be collected when failure happens. The same as current logs collecting logic.

Reply:
Sounds good.